zerver/data_import/mattermost.py

```diff
@@ -311,6 +315,85 @@ def get_mentioned_user_ids(raw_message: Dict[str, Any], user_id_mapper: IdMapper
     return user_ids
 
 
+def process_message_attachments(
```

Comment:
I didn't really understand the use of --mask. Can you please explain what it does?

Reply:
@garg3133 I don't properly remember the full story as well. The description of the flag is this

```
parser.add_argument(
        "--mask",
        dest="masking_content",
        action="store_true",
        help="Mask the content for privacy during QA.",
)
```

I think it was added so that we can import a proper mattermost organization with actual data and do testing on it to make sure that everything is imported correctly. Since we mask the content we won't see the content of the messages during testing.

Reply:
I think that makes sense but it may not be worth the investment; it seems possible that now that the mattermost import tool is pretty stable, we won't use that option again.